content/v1.12/getting-started/provider-azure-part-2.md

```diff
@@ -0,0 +1,1050 @@
+```
+
+_Composite resources_ are great for creating one or more related resources against
+a template, but all _composite resources_ exist at the Kubernetes "cluster
+level." There's no isolation between _composite resources_. Crossplane uses
+_Claims_ to create resources with namespace isolation. 
+
+## Create a claim
+
+_Claims_, just like _composite resources_ use the custom API defined in the
+_XRD_. Unlike a _composite resource_, Crossplane can create _Claims_ in a
+namespace.
+
+### Create a new Kubernetes namespace
+Create a new namespace with `kubectl create namespace`.
+
+```shell
+kubectl create namespace test
+```
+
+Look at the _XRD_ to see the parameters for the _Claim_.
+A _Claim_ uses the same {{<hover label="XRDclaim2" line="6" >}}group{{</hover>}}
+a _composite resource_ uses but a different 
+{{<hover label="XRDclaim2" line="8" >}}kind{{</hover>}}.
+
+```yaml {label="XRDclaim2"}
+apiVersion: apiextensions.crossplane.io/v1
+kind: CompositeResourceDefinition
+# Removed for brevity
+spec:
+# Removed for brevity
+  group: custom-api.example.org
+  claimNames:
+    kind: VirtualMachine
+    plural: virtualmachines
+```
+
+Like the _composite resource_, create a new object with the 
+{{<hover label="claim" line="2" >}}custom-api.example.org{{</hover>}} API
+endpoint.
+
+The _XRD_
+{{<hover label="XRDclaim2" line="8" >}}claimNames.kind{{</hover>}} defines the
+{{<hover label="claim" line="3" >}}kind{{</hover>}}.
+
+The {{<hover label="claim" line="7" >}}spec{{</hover>}} uses the same
+API options as the _composite resource_.
+
+### Apply the claim
+Apply the _claim_ to your Kubernetes cluster.
+
+```yaml {label="claim"}
+cat <<EOF | kubectl apply -f -
+apiVersion: custom-api.example.org/v1alpha1
+kind: VirtualMachine
+metadata:
+  name: claimed-virtualmachine
+  namespace: test
+spec:
+  region: "US"
+EOF
+```
+
+### Verify the claim
+Verify Crossplane created the _claim_ with `kubectl get virtualmachine` in the 
+`test` namespace.
+
+{{<hint "tip" >}}
+View claims with `kubectl get <kind>` or use `kubectl get claim` to view all
+_Claims_.
+{{</hint >}}
+
+```shell {copy-lines="1"}
+kubectl get virtualmachine -n test
+NAME                     SYNCED   READY   CONNECTION-SECRET   AGE
+claimed-virtualmachine   True     True                        3m40s
+```
+
+When Crossplane creates a _Claim_, a unique _composite resource_ is also
+created. View the new _composite resource_ with `kubectl get xvirtualmachine`.
+
+```shell {copy-lines="1"}
+kubectl get xvirtualmachine
+NAME                           SYNCED   READY   COMPOSITION                             AGE
+claimed-virtualmachine-cw6cv   True     True    crossplane-quickstart-vm-with-network   3m57s
+```
+
+The _composite resource_ exists at the "cluster scope" while the _Claim_ exists
+at the "namespace scope."
+
+Create a second namespace and a second claim.
+
+```shell
+kubectl create namespace test2
+cat <<EOF | kubectl apply -f -
```

Comment:
other "here documents" used to directly apply a resource render in syntax highlighted colors, but this one renders in yellow.  is that because of the additional command to create the namespace included in the same block? 🤔 
<img width="473" alt="Screenshot 2023-06-02 at 12 25 42 PM" src="https://github.com/crossplane/docs/assets/4313439/3258c2d6-5ffa-412f-9b23-a513182cbe4f">